openshift/4.0/scripts/oc-sonar-dotnet.sh

```diff
@@ -1,115 +0,0 @@
-#!/bin/bash
-#
```

Comment:
This wasn't being used anywhere in the repo.

Performs a Sonar scan on a .NET Core codebase. 
It cleans up any previous test runs, builds the code, 
runs unit tests with coverage reporting, 
and finally performs the Sonar scan.